src/nomad_measurements/xrd/xrd_parser.py

```diff
@@ -208,10 +221,66 @@ def parse_and_convert_file(file_path):
     '''
     file_path = os.path.abspath(file_path)
 
-    format_parser = FormatParser(file_path)
+    format_parser = FormatParser(file_path, logger)
     parsed_data = format_parser.parse()
 
     data_converter = DataConverter(parsed_data)
     common_data = data_converter.convert()
 
     return common_data
+
+def read_rigaku_RASX(file_path, logger):
+    ''' Reads .rasx files from Rigaku instruments
+        - reader is based on IKZ submodule 
+        - currently supports one scan per file
+        - in case of multiple scans per file, only the first scan is read
+    Args:
+        file_path (string): absolute path of the file
+        logger (object): logger object for propagating errors and warnings
+    Returns:
+        dict: populated with data from Rigaku .RASX files
+    '''
+    reader = RASXfile(file_path)
+    data = reader.get_data()
+    metainfo = reader.get_metainfo()
+    pdata = reader.get_RSM()
+
+    if not metainfo[0]['ScanInformation']['AxisName'] == "TwoTheta":
+        raise NotImplementedError(f"{metainfo[0]['ScanInformation']['AxisName']} scan not supported. Only TwoTheta scan is supported.")

```

Comment:
From discussing with Carsten I think we can remove this check and always treat the axis in the profile data as 2theta.